src/middleware.js

```diff
@@ -356,8 +378,9 @@ export default function autobahnMiddlewareFactory({ connection } = {}) {
         dispatch(connectionOpened(newConnection));
       };
 
-      newConnection.onclose = () => {
+      newConnection.onclose = (reason, details) => {
         autobahnMiddleware._connection = null;
+        dispatch(onDisconnect(reason, details));
```

Comment:
Instead of double dispatching onDisconnect and connectionClosed, could we just add the reason and details to the existing connectionClosed action?

Reply:
Afkors,
I was thinking the same thing before sleep.
I'll fix that.